include/zephyr/arch/riscv/common/linker-tail.ld

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
as a user I can't tell what linker-tail.ld does from the name of the file.

So same comments apply as for the `linker-head.ld.